### ISSUE TITLE
New project action

### ISSUE THREAD

**Issue by KochynVolodymyr** [os-fpga/FOEDAG#218]
Currently we can start new project only once.
Possible solutions:
* Open another instance of the whole FOEDAG project. Create another process.
* Open another main widget within same process.

**Comment by alain-rs** [comment]
The tool should only support one project at a time. 
If a new project is needed, prompt the user to close the previous project, then open the new project.
Within one project, we can support multiple compilation runs with different compilation options, but only one project.

**Comment by ravic-rs** [comment]
@KochynVolodymyr I believe we have a solution for this here https://github.com/os-fpga/FOEDAG/pull/549
Please confirm.

**Comment by KochynVolodymyr** [comment]
> @KochynVolodymyr I believe we have a solution for this here #549 Please confirm.

No. We have another solution - Close project. But it is kind of weird behavior and out of user experience.
